modules/_labs/color-picker/react/lib/ColorPicker.tsx

```diff
@@ -29,14 +29,19 @@ export interface ColorPickerProps extends React.HTMLAttributes<HTMLDivElement> {
    */
   showCustomHexInput?: boolean;
   /**
-   * The label text of the custom hex input. This is also used as the aria-label
+   * The label text of the custom hex input.
    * @default 'Custom Hex Color'
    */
   customHexInputLabel?: string;
+  /**
+   * The label of the custom hex color submit icon button.
+   * @default 'Submit'
+   */
+  submitLabel?: string;
   /**
    * The function called when the submit icon is clicked.
    */
-  onSubmitClick?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
+  onSubmitClick?: (event: React.FormEvent) => void;
```

Comment:
This change should also be reflected in the README (duplicate documentation strikes again).